submodules/linphone

```diff
@@ -1 +1 @@
-Subproject commit 82e395bc9b140f7f97b7ab9b41d8e5aa41f70965
+Subproject commit 0e996357d70a606a86d97af7d28eaa8451f09952
```

Comment:
The issue is that  h263  isn't enabled in the current build and that need to be fixed..
See commit for the fix.
https://github.com/BelledonneCommunications/mediastreamer2/commit/863a2d612832cf8722ce345246156baba9cc93be